src/main/kotlin/Stores/TeamStore.kt

```diff
@@ -0,0 +1,37 @@
+package Stores
+
+import io.ktor.http.Parameters
+import models.Team
+import models.Team.createdAt
+import models.Team.id
+import models.Team.name
+import models.Team.updateAt
+import org.jetbrains.exposed.sql.select
+import org.jetbrains.exposed.sql.update
+import org.joda.time.DateTime
+
+data class TeamObj (
+        var id: String,
+        var name: String,
+        var createdAt: String,
+        var updateAt: String
+)
+
+object TeamStore : BaseStore<Team>(Team) {
+    suspend fun create(params: Parameters): String? {
+        return create  {
```

Comment:
extra space here

Reply:
also let's change this to more specifically `createTeam()`. Might be obvious, but just in case